compass-server/pom.xml

```diff
@@ -25,11 +25,10 @@
 
         <start-class>org.zalando.compass.Application</start-class>
 
-        <spring-boot.version>1.4.1.RELEASE</spring-boot.version>
+        <spring-boot.version>1.4.2.RELEASE</spring-boot.version>
+        <jackson.version>2.8.4</jackson.version>
+        <flyway.version>3.2.1</flyway.version>
```

Comment:
I haven't found a way to reference properties of an *imported* `pom`, seems like this is only possible for parent-poms.